conanfile.py

```diff
@@ -196,23 +199,26 @@ def package_info(self):
             self.cpp_info.includedirs.append(os.path.join(self.build_folder, "include"))
             bindir = os.path.join(self.build_folder, "bin")
             luadir = os.path.join(self.source_folder, "engine/lua")
-            pydir = os.path.join(self.source_folder, "python/python_api")
+            pydir = os.path.join(self.build_folder, "lib/cloe/python")
+            py_path = os.path.join(self.source_folder, "python/python_api")
             libdir = os.path.join(self.build_folder, "lib")
         else:
             self.cpp_info.builddirs.append(os.path.join("lib", "cmake", "cloe"))
             bindir = os.path.join(self.package_folder, "bin")
             luadir = os.path.join(self.package_folder, "lib/cloe/lua")
             pydir = os.path.join(self.package_folder, "lib/cloe/python")
+            py_path = pydir
             libdir = None
 
         self.output.info(f"Appending PATH environment variable: {bindir}")
         self.runenv_info.prepend_path("PATH", bindir)
         self.output.info(f"Appending CLOE_LUA_PATH environment variable: {luadir}")
         self.runenv_info.prepend_path("CLOE_LUA_PATH", luadir)
         if libdir is not None:
+            #TODO
             self.output.info(f"Appending LD_LIBRARY_PATH environment variable: {libdir}")
             self.runenv_info.append_path("LD_LIBRARY_PATH", libdir)
         if self.options.python_api:
-            self.output.info(f"Appending PYHTONPATH and CLOE_PYTHON_BINDINGS environment variables: {pydir}")
-            self.runenv_info.prepend_path("PYTHONPATH", str(pydir))
-            self.runenv_info.prepend_path("CLOE_PYTHON_BINDINGS", str(pydir))
+            self.output.info(f"Appending PYHTONPATH and BASIC_CLOE_PYTHON_BINDINGS environment variables: {pydir}")
+            self.runenv_info.prepend_path("PYTHONPATH", str(py_path))
+            self.runenv_info.prepend_path("BASIC_CLOE_PYTHON_BINDINGS", str(pydir))
```

Comment:
Why are you renaming `CLOE_PYTHON_BINDINGS` which is a generic construct to `BASIC_CLOE_PYTHON_BINDINGS`?

Reply:
I renamed it to the original